### PR TITLE
example of refactored language competency levels

### PR DESCRIPTION
**Pull request by wafer-bw**
An example of my suggestion to lower the level 1 requirements of language competencies maintaining the worth of the original competency in level 2.

### CODE REVIEW

File: competencies/language-python.md [hunk 1]
```diff
@@ -1,44 +1,37 @@
...
+* You can describe the kinds of problem the language solves very well and the problems it's not a great fit for.
+* You understand packaging paradigms for the language for both publishing and dependency management.
+* You understand how to review third party libraries lifecycles and security implications.
 
-* Pip (and other package managers)
+## How do you improve it?
+* Open many PRs
+* Syntax conventions: Read the PEP8 style guide: [https://www.python.org/dev/peps/pep-0008/](https://www.python.org/dev/peps/pep-0008/)
 
+## Key points of research
```
Comment (codeallthethingz) [inline competencies/language-python.md:32]: Can we add pipenv it's my favourite package manager / python version system.

Comment (wafer-bw) [reply competencies/language-python.md:32]: added!

File: competencies/language-python.md [hunk 2]
```diff
@@ -1,44 +1,37 @@
...
-* Lambdas
-
-* List and dictionary comprehensions
-
-* async/await
+# Competency - Language Python - Level 2
 
-* Global interpreter lock
+## How do you prove it?
+* For code you have submitted, your code reviews get little syntactic corrections.
```
[the remark this answers is not in the document]
Comment (wafer-bw) [reply competencies/language-python.md]: updated

File: competencies/language-python.md [hunk 3]
```diff
@@ -1,44 +1,37 @@
 # Competency - Language Python
```
[the remark this answers is not in the document]
Comment (wafer-bw) [reply competencies/language-python.md]: updated

File: competencies/language-python.md [hunk 4]
```diff
@@ -1,44 +1,37 @@
...
-* List and dictionary comprehensions
-
-* async/await
+# Competency - Language Python - Level 2
 
-* Global interpreter lock
+## How do you prove it?
+* For code you have submitted, your code reviews get little syntactic corrections.
+* You can write code from memory without having to look up how you use language specific features.
+* You can describe the kinds of problem the language solves very well and the problems it's not a great fit for.
```
[the remark this answers is not in the document]
Comment (wafer-bw) [reply competencies/language-python.md]: updated

File: competencies/language-python.md [hunk 5]
```diff
@@ -1,44 +1,38 @@
...
-
-Read the PEP8 style guide: [https://www.python.org/dev/peps/pep-0008/](https://www.python.org/dev/peps/pep-0008/)
+* Do Google's python class: [https://developers.google.com/edu/python/](https://developers.google.com/edu/python/)
+* Write a lot of code
 
 ## Key points of research
-
 * Differences between Python 2 and 3
+* Pip (and other package managers)
+* Third party libaries
```
Comment (grrilla) [inline competencies/language-python.md:17]: Are there notable third party libraries that are considered core by the Python community?

Comment (wafer-bw) [reply competencies/language-python.md:17]: Depends what you're doing. I'll add some suggestions that are very common.

**Pull request merged**